rust/ranked-prolly-tree/README.md

```diff
@@ -0,0 +1,153 @@
+# ranked-prolly-tree
+
+
+`ranked-prolly-tree` (RPT) is a key-value store implemented as a [prolly tree], utilizing a flexible content-addressed block storage backend. RPT is designed to be the foundation of a lazy database, utilizing partial sync on demand.
```

Comment:
```suggestion
`ranked-prolly-tree` (RPT) is a key-value store implemented as a [prolly tree], utilizing a flexible content-addressed block storage backend. RPT is designed to be the foundation of a passive database, providing on demand partial replication.
```

1. I think "passive" is better term here because it has no component that actively participates in replication. Lazy on the other hand often implies deferred action occurring on read/write. I suppose querying replica can be considered lazy, while source replica is passive.

2. I think using replication is more accurate term here than sync, as later to me implies bidirectional replication which isn't necessarily implied here.